AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyScriptCanvasVariableManager.py

```diff
@@ -71,8 +72,44 @@ def create_new_variable(self, new_variable_type: str) -> None:
         table_view = self.variable_manager.findChild(QtWidgets.QTableView, VARIABLE_PALETTE_QT)
         variable_entry = pyside_utils.find_child_by_pattern(table_view, new_variable_type)
 
-        # Click on it to create variable
+        # todo
+        # maybe put these 3 steps into a helper function
+        # maybe assert here ?
```

Comment:
Make these decisions and either implement them or create GHI to track the backlog work.

Reply:
this was mistakenly left in. the validation comes immediately after you call the function